### PR TITLE
[9.0] Upgrade Partner Gender module

### PR DESCRIPTION
**Pull request by richard-willdooit**

### CODE REVIEW

File: partner_contact_gender/__openerp__.py [hunk 1]
```diff
@@ -18,18 +17,22 @@
 
 {
     "name": "Contact gender",
-    "version": "8.0.1.0.0",
-    "category": "Customer Relationship Management",
-    "author": "Odoo Community Association (OCA), Grupo ESOC",
+    "summary": "Add gender field to contacts",
+    "version": "9.0.1.0.0",
+    "author": "Odoo Community Association (OCA)",
```
Comment (elicoidal) [inline partner_contact_gender/__openerp__.py]: Why remove previous author? why not add Willow-IT?

File: partner_contact_gender/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline partner_contact_gender/models/__init__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: partner_contact_gender/__openerp__.py (hunk 1 above)
Comment (richard-willdooit) [inline partner_contact_gender/__openerp__.py]: Happy to add original back, but I would have thought if all authors were listed, that would make it very long and unweildy - I thought that was the purpose of contributors, to some degree...

File: partner_contact_gender/models/__init__.py (hunk 2 above)
Comment (richard-willowit) [inline partner_contact_gender/models/__init__.py]: These are the simple headers?  Sorry, I think I am missing something.

File: partner_contact_gender/__openerp__.py [hunk 3]
```diff
@@ -1,31 +1,21 @@
...
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (yvaucher) [inline partner_contact_gender/__openerp__.py:3]: Copyright line is necessary don't remove it please

File: partner_contact_gender/models/partner_gender.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (yvaucher) [inline partner_contact_gender/models/partner_gender.py]: Missing copyright line

File: partner_contact_gender/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (yvaucher) [inline partner_contact_gender/models/__init__.py:3]: Missing copyright line

File: partner_contact_gender/models/partner_gender.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+from openerp import fields, models, _, api
```
Comment (yvaucher) [inline partner_contact_gender/models/partner_gender.py]:
```
partner_contact_gender/models/partner_gender.py:4:1: F401 '_' imported but unused
partner_contact_gender/models/partner_gender.py:4:1: F401 'api' imported but unused
```

**Comment by yvaucher** [comment]
Just a thought: I wonder if it would be possible to auto complete the title from gender if not set or reversely.

The same selection could appear on title with a boolean to know the default title to set for Male or Female.

For now you can just fix my remarks and it will be good.

Thanks for porting it.

**Comment by gurneyalex** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**